.github/workflows/test.yml

```diff
@@ -0,0 +1,75 @@
+name: Test
```

Comment:
lint.yml and test.yml should be merged into one workflow that should also run for pull requests. pr-check.yml duplicates too much of the other workflows, it should be merged with the workflow that unifies lint.yml and test.yml